Albo1125.Common/CommonLibrary/ExtensionMethods.cs

```diff
@@ -449,6 +449,26 @@ public static void SetColors(this Vehicle v, VehicleColor color)
         }
 
         /// <summary>
+        /// Randomise the license plate to avoid excessively frequent debug plates from showing.
+        /// </summary>
+        public static void RandomiseLicencePlate(this Vehicle vehicle)
+        {
+            if (vehicle)
+            {
+                vehicle.LicensePlate = MathHelper.GetRandomInteger(9).ToString() +
```

Comment:
Considering we are doing this with realism in mind, I took a look around and found [this page](https://www.dmv.ca.gov/portal/dmv/?1dmy&urile=wcm:path:/dmv_content_en/dmv/pubs/plates/standardplates) with details on how at least CA generates their license plate numbers. At the very least, removing `I`, `O` and `Q` will help with minimizing ambiguous license plates. We can also generate different plates based on [the class of the vehicle](http://docs.ragepluginhook.net/html/T_Rage_VehicleClass.htm).

The following complies with the specification above plates - [**Example on DotNetFiddle**](https://dotnetfiddle.net/tGFY7N) 

```c#
	public static string GeneratePassengerLicensePlateNumber()
	{
		return gen.Next(10, 99).ToString() + 
			ValidPassengerCharacters[gen.Next(0, ValidPassengerCharacters.Length)] + 
			ValidPassengerCharacters[gen.Next(0, ValidPassengerCharacters.Length)] + 
			ValidPassengerCharacters[gen.Next(0, ValidPassengerCharacters.Length)] + 
			gen.Next(100, 999).ToString();
	}
	
	public static string GenerateCommercialLicensePlateNumber()
	{
		return gen.Next(10, 99).ToString() + 
			ValidCommercialCharacters[gen.Next(0, ValidCommercialCharacters.Length)] + 
			gen.Next(10000, 99999).ToString();
	}
	
	public static string GenerateMotorcycleLicensePlateNumber()
	{
		return gen.Next(10, 99).ToString() + 
			ValidPassengerCharacters[gen.Next(0, ValidPassengerCharacters.Length)] + 
			gen.Next(1000, 9999).ToString();
	}
	
	public static string GenerateTrailerLicensePlateNumber()
	{
		return gen.Next(10, 99).ToString() + 
			ValidPassengerCharacters[gen.Next(0, ValidPassengerCharacters.Length)] + 
			ValidPassengerCharacters[gen.Next(0, ValidPassengerCharacters.Length)] + 
			gen.Next(1000, 9999).ToString();
	}
```

Reply:
Appreciate the input. However, GTA5 default plates follow the format as per @LSPDFR-PeterU code. I find it more appropriate for a randomizer to follow the default in-game format rather than force it onto Canadian - a Canadian number plate generator could be an interesting idea for a separate plugin (so would a British one? :P )